packages/strapi-plugin-upload/admin/src/components/ModalTab/index.js

```diff
@@ -22,24 +26,36 @@ const ModalTab = ({ isDisabled, label, to, isActive, onClick }) => {
   };
 
   return (
-    <Wrapper isActive={isActive} isDisabled={isDisabled} onClick={handleClick}>
-      <FormattedMessage id={getTrad(`modal.nav.${label}`)} />
-    </Wrapper>
+    <Padded right size="lg">
+      <Wrapper isActive={isActive} isDisabled={isDisabled} onClick={handleClick}>
+        <Flex>
+          <IntlText
+            id={getTrad(`modal.nav.${label}`)}
+            textTransform="uppercase"
+            fontWeight={isActive ? 'bold' : 'semiBold'}
+            color={isActive ? 'mediumBlue' : 'grey'}
+          />
+          {typeof count === 'number' && <Count count={count} />}
```

Comment:
`count` propType is set as number, so can it be something else?

Reply:
Count can be null.